packages/components/src/popover/index.js

```diff
@@ -262,6 +263,21 @@ const Popover = ( {
 	const [ containerResizeListener, contentSize ] = useResizeObserver();
 	noArrow = isExpanded || noArrow;
 
+/* 	function selector( select ) {
+		const {
+			x,
+		} = select( store );
+		return {
+			x: isToolBarActive(),
+		};
+	}
+
+	let {
+		x,
+	} = useSelect( selector, [] );
+	x = x || 12
+	console.log(x); */
```

Comment:
pushing commented code is generally not a good idea as long as it is not for the documentation purposes